client/my-sites/domains/domain-management/list/domain-only.jsx

```diff
@@ -43,6 +44,11 @@ const DomainOnly = ( { primaryDomain, hasNotice, recordTracks, siteId, slug, tra
 
 	return (
 		<div>
+			<PageViewTracker
+				path="/view/:domain"
+				properties={ { site: 'no-site' } }
```

Comment:
```suggestion
				path="/view/:site"
```

This isn't necessarily a problem, but I was curious to see why we had to specify `properties={ { site: 'no-site' } }` here even though the same data is also added in `getSuperProps`. 

Turns out that this is because `shouldReportOmitBlogId` thinks that because we are using the slug `/view/:domain`, we shouldn't include data about the current blog in the Tracks event data. However, the "real" slug that renders this page is `/view/:site` ([ref](https://github.com/Automattic/wp-calypso/blob/add/domain-only-tracking/client/my-sites/preview/index.js#L7)) and if we were to use that string as the `path` for the Tracks event, we wouldn't have to explicitly include the `{ site: 'no-site' }` prop here.

I'm open for other arguments, but instinctively I think it makes more sense to pass `path="/view/:site"` here instead.